extensions/amp-lightbox/1.0/storybook/Basic.amp.js

```diff
@@ -57,6 +57,291 @@ export const Default = () => {
   );
 };
 
+export const scrollable = () => {
+  const animation = select('animation', [
+    'fade-in',
+    'fly-in-top',
+    'fly-in-bottom',
+  ]);
+  const backgroundColor = text('background color', 'rgba(0,0,0,0.5)');
+  const color = text('font color', '');
+  const scrollable = boolean('scrollable', true);
+  const lotsOfText = boolean('lots of text?', true);
+  return (
+    <>
+      <style>{`
+        #lightbox {
+          background-color: ${backgroundColor};
+          color: ${color};
+        }
+      `}</style>
+      <div style="height: 300px;">
+        <amp-lightbox
+          id="lightbox"
+          layout="nodisplay"
+          animation={animation}
+          scrollable={scrollable}
```

Comment:
I don't see the attribute mapping anywhere.

Reply:
You mean in `amp-lightbox.js`? It was already there:
https://github.com/ampproject/amphtml/blob/cb907bee87bf1de15654e3afd6c899ba6815dde3/extensions/amp-lightbox/1.0/amp-lightbox.js#L102